src/traces/parcoords/base_plot.js

```diff
@@ -27,18 +26,25 @@ exports.clean = function(newFullData, newFullLayout, oldFullData, oldFullLayout)
 
 exports.toSVG = function(gd) {
 
-    var canvas = document.querySelector('.parcoords-lines');
+    function canvasToImage(canvas) {
+        var rect = canvas.getBoundingClientRect();
+        var compStyle = window.getComputedStyle(canvas, null);
+        var canvasContentOriginX = parseFloat(compStyle.getPropertyValue('padding-left')) + rect.left;
+        var canvasContentOriginY = parseFloat(compStyle.getPropertyValue('padding-top')) + rect.top;
+
+        var imageData = canvas.toDataURL('image/png');
+        var image = gd._fullLayout._glimages.append('svg:image');
+        image.attr({
+            xmlns: xmlnsNamespaces.svg,
+            'xlink:href': imageData,
+            x: canvasContentOriginX,
+            y: canvasContentOriginY
+        });
+    }
 
-    var rect = canvas.getBoundingClientRect();
+    var canvases = document.querySelectorAll('.parcoords-lines.context, .parcoords-lines.focus');
 
-    var imageData = canvas.toDataURL('image/png');
-    var image = gd._fullLayout._glimages.append('svg:image');
-    image.attr({
-        xmlns: xmlnsNamespaces.svg,
-        'xlink:href': imageData,
-        x: -rect.left,
-        y: parseFloat(window.getComputedStyle(canvas, null).getPropertyValue('padding-top')) + rect.top
-    });
+    canvases.forEach(canvasToImage);
```

Comment:
Yes, it still does, as I'm optimizing for feature completion so it's possibly mergeable this week. When you're away, I can unify the canvases (it's not a big deal) and rework the test cases based on your earlier feedback. Btw. I'll speed up the test cases in the short term, now they use 2k data points which is way too many.